cmd/monaco/account/deploy.go

```diff
@@ -121,9 +121,13 @@ func deploy(fs afero.Fs, opts deployOpts) error {
 		return errors.New("failed to load all account management resources")
 	}
 
+	if opts.dryRun {
+		return nil
```

Comment:
I think it'd make sense to log something similar to the config dry-run. E.g. "Successfully validated account resources"